homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -54,18 +54,23 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the Waze travel time sensor platform."""
-    destination = config.get(CONF_DESTINATION)
-    name = config.get(CONF_NAME)
-    origin = config.get(CONF_ORIGIN)
-    region = config.get(CONF_REGION)
-    incl_filter = config.get(CONF_INCL_FILTER)
-    excl_filter = config.get(CONF_EXCL_FILTER)
-
-    sensor = WazeTravelTime(name, origin, destination, region,
-                            incl_filter, excl_filter)
-
-    add_devices([sensor], True)
```

Comment:
The problem is probably that we call update during entity addition and then the call to update when home assistant start event is fired is throttled.

So the fix is to just remove the update during entity addition, ie remove `True` here. Don't make any other changes.